apps/lend/src/pages/index.tsx

```diff
@@ -35,7 +35,7 @@ const App: NextPage = () => {
       {SubRoutes}
       <Route path=":locale">{SubRoutes}</Route>
       <Route path="/markets/*" element={<Navigate to={`/ethereum${ROUTE.PAGE_MARKETS}`} replace />} />
-      <Route path="/risk-disclaimer" element={<Navigate to={`/ethereum${ROUTE.PAGE_RISK_DISCLAIMER}`} replace />} />
```

Comment:
do we need to set up a redirect for old links living elsewhere?

Reply:
Oh I just replaced all the old links (I could find) with the new one